packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -494,6 +494,13 @@ class FileCollection extends DocumentCollection {
     forceFileDownload(`${href}?Dl=1`, filenameToUse)
   }
 
+  async fetchFileContent(id) {
+    console.info(
```

Comment:
Please use console.warn. This way if we use it in an app that has configured its tests to break on warn, they will get notified and will be able to change the call.